CHANGELOG.md

```diff
@@ -16,7 +16,9 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 - Update the `forbidden-io-function-checker` to check functions from imported modules as well as methods (according to their qualified name)
 - Update the `forbidden-io-function-checker` to flag aliases of forbidden functions
 - Update how error messages are overridden such that section headers are no longer required within the config file
-- Added `presistent_server` which recives the watch property changes through websockets and updates the HTML report
+- Added `presistent_server` which receives the watch property changes through websockets and updates the HTML report
+- Added test case to `test_check.py` for better coverage
```

Comment:
For this entry in particular, describe what situation is being tested (i.e., what kind of input to `check_all`)